tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/instance_method_tests.bal

```diff
@@ -191,3 +200,15 @@ function newByte(int val) returns handle = @java:Constructor {
    'class: "java.lang.Byte"
 } external;
 
+const ASSERTION_ERROR_REASON = "AssertionError";
+
+function assertEquality(any|error expected, any|error actual) {
```

Comment:
Since this is unit tests we can use the `test:assertEquals` function from the `ballerina/test` package. Then we need not define this function